lib/re_captcha/configuration.rb

```diff
@@ -33,7 +36,22 @@ def language_code(locale)
     end
 
     def valid?
-      !private_key.nil? && !public_key.nil?
+      reset_errors
+      set_errors
+      errors.empty?
+    end
```

Comment:
Using ActiveModel means adding ActiveSupport and all their requirements.  It looks overkill to me.  This is going to slow down the load time in order to clean a couple of lines of code.  My point is that this gem is intended to be used with any Ruby app (not just Rails).